draft-ietf-tls-rfc4492bis.xml

```diff
@@ -339,7 +340,10 @@
             deprecates all but the uncompressed point format. Implementations of this document MUST support the
             uncompressed format for all of their supported curves, and MUST NOT support other formats for curves
             defined in this specification. For backwards compatibility purposes, the point format list extension
-            MUST still be included, and contain exactly one value: the uncompressed point format (0).</t>
+            MAY still be included, and contain exactly one value: the uncompressed point format (0). RFC 4492 
```

Comment:
Is that necessary?  "MAY still be included and if so MUST contain" reads very clunky.